src/main/SnakeAndLadders/Practice/Model/Players.java

```diff
@@ -0,0 +1,27 @@
+package SnakeAndLadders.Practice.Model;
+
+public class Players {
```

Comment:
I think this entity is for a single player and should be named Player instead of Players